python/dgl/convert.py

```diff
@@ -1830,7 +1833,7 @@ def to_networkx(
         return _to_networkx_homogeneous(g, node_attrs, edge_attrs)
```

Comment:
Also do this for homogeneous graphs?

Reply:
Sure, I can add this, but the test for the homogeneous (current) version is still disabled. I'd recommend to leave this out to leave the existing code as unchanged as possible, so that it's easy to make changes later if needed.

Reply:
Of course. Done.